app/src/ActionModal/components/DelegationModal.vue

```diff
@@ -355,6 +355,10 @@ export default {
 
       // update registered topics for emails as the validator set changed
       this.$store.dispatch("updateEmailRegistrations")
+      // update the role of the user as it might change after bonding the first time
+      if (this.currentNetwork.network_type === "polkadot") {
+        this.$store.dispatch("checkAddressRole")
```

Comment:
```suggestion
     await this.$store.dispatch("checkAddressRole")
```

Reply:
why await?

Reply:
ok, you want it non-blocking. Makes sense